docs/release_notes/0.128.0.md

```diff
@@ -0,0 +1,24 @@
+# Release v0.128.0
+
+## 🎯 Improvements
+
+- Improve unit test coverage of create cluster - nodegroups (#6177)
```

Comment:
This one is a duplicate

Reply:
It was merged two days ago, I think you're thinking of `Improve unit test coverage of create cluster - Karpenter (https://github.com/weaveworks/eksctl/pull/6143)`.